client/gutenberg/extensions/contact-info/address/edit.js

```diff
@@ -0,0 +1,95 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { PlainText } from '@wordpress/editor';
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import save from './save';
+
+class AddressEdit extends Component {
+	constructor( ...args ) {
+		super( ...args );
+
+		this.preventEnterKey = this.preventEnterKey.bind( this );
```

Comment:
do we need to do this?

Reply:
yes Otherwise we end up with a new line. 
I think the work around there might be to use the RichText instead